### PR TITLE
Correct typos and re-word Smoothing step in ROI mask recipe

### PR DESCRIPTION
**Pull request by reiningc**
Fix for #2058 
Also reviewed the rest of the doc and fixed a couple other typos I noticed.

Let me know if you need me to change anything!

### CODE REVIEW

**Comment by jeromedockes** [comment]
thanks! LGTM

File: doc/manipulating_images/manipulating_images.rst [hunk 1]
```diff
@@ -198,7 +198,7 @@ brain. It is thus convenient to apply a brain mask in order to convert the
     :width: 100%
 
 Note that in an analysis pipeline, this operation is best done using the
-:ref:`masker objects <masker_objects>`. For completness, we give code to
+:ref:`masker objects <masker_objects>`. For completeness, we give code to
```
Comment (kchawla-pi) [inline doc/manipulating_images/manipulating_images.rst]: ...we give the code to do it manually, below...

**Comment by kchawla-pi** [comment]
Thanks so much! Very cool! Gave a comment. Fix that and we can merge.

File: doc/manipulating_images/manipulating_images.rst (hunk 1 above)
Comment (reiningc) [reply doc/manipulating_images/manipulating_images.rst]: Can do!

**Comment by jeromedockes** [comment]
thanks!
+1 for merge

**Comment by GaelVaroquaux** [comment]
Thanks! Merging!

**Pull request merged**